samples/login-sample/src/main/java/com/uber/sdk/android/samples/LoginSampleActivity.java

```diff
@@ -127,14 +140,35 @@ protected void onCreate(Bundle savedInstanceState) {
                 new SampleLoginCallback(),
                 configuration,
                 CUSTOM_BUTTON_REQUEST_CODE);
-
         customButton = (Button) findViewById(R.id.custom_uber_button);
         customButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 loginManager.login(LoginSampleActivity.this);
             }
         });
+        appLinkButton = findViewById(R.id.applink_uber_button);
+        appLinkButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(Intent.ACTION_VIEW);
+                String codeChallenge = "";
+                try {
+                    codeChallenge = PKCEUtil.generateCodeChallange(CODE_VERIFIER);
+                } catch (UnsupportedEncodingException | NoSuchAlgorithmException e) {
+                    throw new RuntimeException(e);
+                }
+                intent.setData(
+                        AuthUriAssembler.assemble(
+                                        ResponseType.CODE,
+                                        loginManager.getSessionConfiguration(),
+                                        codeChallenge,
+                                        null
+                                )
+                );
+                startActivityForResult(intent, CUSTOM_BUTTON_REQUEST_CODE);
```

Comment:
Do we need new explicit handling for onActivityResult if you're using a new call here?